src/components/forms/Register.js

```diff
@@ -203,10 +203,10 @@ const Register = () => {
   );
 };
 
-const LoginP = styled.p`
-    text-align: center;
-    font-family: 'Nunito';
-`
+// const LoginP = styled.p`
```

Comment:
Unless you might be using these again remove the commented out code. If it's being saved for later add a note about why it's commented out. 👍

Reply:
I was thinking of doing a code cleanup branch for RC 1 once everything is done.

Reply:
Sounds good. There should be plenty of time for that tomorrow based on how much got done today. 💯